src/index.html

```diff
@@ -11,8 +11,105 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="card__photo"
+        />
+
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <p class="card__price">
+          <span class="card__price-title">Price:</span>
+
+          <span class="card__price-value">$2,199</span>
+        </p>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="hover"
+        >
+          BUY
+        </a>
+      </article>
+    </div>
+
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        class="card__photo"
+      />
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__reviews">Reviews: 5</p>
+      </div>
+
+      <p class="card__price">
+        <span class="card__price-title">Price:</span>
+
+        <span class="card__price-value">$2,199</span>
+      </p>
+
+      <a
+        href="#"
+        class="card__button"
+        data-qa="hover"
+      >
+        BUY
+      </a>
+    </article>
```

Comment:
I think you accidentally duplicated this code. 

![image](https://github.com/mate-academy/layout_product-cards/assets/54860641/a7b85b04-2b12-4e2e-bf64-c42ea474041d)